### PR TITLE
add more issues types

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

requested by @williamFalcon 

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7

### CODE REVIEW

**Review by rohitgr7 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-  - name: Ask a Question
+  - name: ❓ Ask a Question
     url: https://github.com/Lightning-AI/lightning/discussions/new
-    about: Ask and answer Lightning related questions
-  - name: 💬 Slack
+    about: Ask and answer Lightning related questions.
+  - name: 💬 Chat with us
     url: https://www.pytorchlightning.ai/community
-    about: Chat with our community
+    about: Live chat with Prefect experts, engineers, and users on our Slack community.
```
Comment (manskx) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Live chat with experts, engineers, and users on our Slack community.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+    about: Ask and answer Lightning related questions.
+  - name: 💬 Chat with us
     url: https://www.pytorchlightning.ai/community
-    about: Chat with our community
+    about: Live chat with experts, engineers, and users on our Slack community.
+  - name: 📖 Read the documentation
+    url: https://lightning.ai/lightning-docs/
+    about: Please consult the documentation before opening any issues!
+  - name: 🙋 Contact us about professional services
+    url: https://lightning.ai
```
Comment (kaushikb11) [inline .github/ISSUE_TEMPLATE/config.yml:13]: We should have a contact us option on `lightning.ai`. The standalone URL won't help the user out much.

**Review by kaushikb11 — APPROVED** [review]

**Review by manskx — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (Borda) [reply .github/ISSUE_TEMPLATE/config.yml:13]: I agree, but there is only pop-up window now...
cc: @edenlightning @lantiga

Comment (awaelchli) [reply .github/ISSUE_TEMPLATE/config.yml:13]: Me too, the other day I searched for several minutes for an email or contact form. There is nothing to be found xD

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
-  - name: Ask a Question
+  - name: ❓ Ask a Question
     url: https://github.com/Lightning-AI/lightning/discussions/new
-    about: Ask and answer Lightning related questions
-  - name: 💬 Slack
+    about: Ask and answer Lightning related questions.
+  - name: 💬 Chat with us
     url: https://www.pytorchlightning.ai/community
-    about: Chat with our community
+    about: Live chat with experts, engineers, and users on our Slack community.
```
Comment (awaelchli) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Live chat with experts, engineers, and users in our Slack community.
```

**Review by awaelchli — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
+  - name: 📖 Read the documentation
+    url: https://lightning.ai/lightning-docs/
+    about: Please consult the documentation before opening any issues!
+  - name: 🙋 Contact us about professional services
```
[the remark this answers is not in the document]
Comment (Felonious-Spellfire) [reply .github/ISSUE_TEMPLATE/config.yml:12]: @awaelchli 
The choice was made to keep things informal and conversational. It's a style choice.

**Review by akihironitta — APPROVED** [review]

**Review by edenlightning — APPROVED** [review]

**Pull request merged**